lib/osi-licenses.yml

```diff
@@ -0,0 +1,54 @@
+# Generated by ChatGPT on 2023-11-18
```

Comment:
This provenance is helpful!

Reply:
Well, it really came from https://opensource.org/licenses/?categories=popular-strong-community, which I had ChatGPT summarize for me as yaml. So that OSI link is a better reference. This was really a note to myself.